src/tasks/withdrawService.ts

```diff
@@ -208,8 +208,12 @@ export async function withdrawAndDump({
   ).flat();
 
   console.log("Recovering list of tokens traded since the previous run...");
+  // Add extra blocks before the last update in case there was a reorg and new
+  // transactions were included.
+  const maxReorgDistance = 20;
```

Comment:
That's a pretty big value. Wouldn't 5 or 10 also be fine?

Reply:
The cost of keeping it high is really minimal. I don't know the size of the longest reorg however.

Reply:
I'm OK with the high value. We used 25 in GPv1. Higher is safer, and the worse that can happen is that we just won't consider some very recently traded tokens until the next run - so very minimal impact.